### ISSUE TITLE
Multidimensional array functionality for Session class

### ISSUE THREAD

**Issue by bobdesaunois** [bobdesaunois/emmaphp#93]
As it is you can only store variables in the Session class with regular variables.
There should be functionality for multidimensional arrays as such:

``
Session::setMulti ("order", "orderMulti"...);
``

It should accept any length of variables.

**Comment by NathanBastiaans** [comment]
This is how we ( or just me ) do it now.
```
$array = array ( 
    array('key' => $val, 'key2', $val), 
    array('key' => $val, 'key2', $val) 
);

Session::set ( 'myMultiArray', $array );
```

**Comment by bobdesaunois** [comment]
@NathanBastiaans Some users just aren't that "agile".

**Comment by NathanBastiaans** [comment]
@bobdesaunois But then again, some users manage to get memory warnings out of a simple SQL statement...

**Comment by bobdesaunois** [comment]
TMFRHHHHHHHHHHH
UYBGHDEXWXSWZSSXWXSWXSWSXWSEWXSWDWSXSWXSWSWDSWXWWDSXSXWS

**Comment by kveler** [comment]
DJKSAHDSAHDKJSAJHASDGFASHDGFASDFGASHFGAHSDFGALSDFGLHASDF

Is dat wat je bedoelde?

**Comment by bobdesaunois** [comment]
Nee, ik bedoelde DJKSAHDSAHDKJSAJHASDGFASHDGFASDFGASHFGAHSDDDDDDDDDDDD
